src/GeoExt/selection/FeatureModel.js

```diff
@@ -135,13 +135,14 @@ Ext.define('GeoExt.selection.FeatureModel', {
 
     /**
      * Called after this.grid is defined.
-     * 
+     *
      * @private
      */
     bindComponent: function() {
         this.callParent(arguments);
         if (this.layerFromStore) {
-            var layer = this.view.getStore() && this.view.getStore().layer;
+            var view = this.view || this.views[0],
+                layer = view.getStore() && view.getStore().layer;
```

Comment:
I know this was like this already, but I really think we should if else here instead of &&